include/linker/common-rom.ld

```diff
@@ -41,9 +41,9 @@
 	{
 		__device_handles_start = .;
 #ifdef LINKER_PASS2
-		KEEP(*(SORT(.__device_handles_pass2)));
+		KEEP(*(SORT(.__device_handles_pass2*)));
 #else /* LINKER_PASS2 */
-		KEEP(*(SORT(.__device_handles_pass1)));
+		KEEP(*(SORT(.__device_handles_pass1*)));
```

Comment:
Pedantry: if we ever get more than 9 passes, this will break by matching __device_handles_pass10.